### PR TITLE
.bat: ignore tasklists localization strings

### PR DESCRIPTION
**Pull request by schmidthubert**

### CODE REVIEW

File: media-autobuild_suite.bat [hunk 1]
```diff
@@ -22,7 +22,7 @@
 
 @echo off
 color 80
-title media-autobuild_suite
+title MABSbat
```
Comment (wiiaboo) [inline media-autobuild_suite.bat]: Move this title change closer to where it's needed.

File: media-autobuild_suite.bat [hunk 2]
```diff
@@ -1552,7 +1551,7 @@ start /I %instdir%\%msys2%\usr\bin\mintty.exe -i /msys2.ico -t "media-autobuild_
 --rav1e=%rav1e% --ripgrep=%ripgrep% --dav1d=%dav1d%'
 
 endlocal
-if [%forceQuitBatch%]==[y] taskkill /pid %ourPID% /f
+if [%forceQuitBatch%]==[y] taskkill /f /pid %ourPID% /f
```
Comment (wiiaboo) [inline media-autobuild_suite.bat]: There's already `/f` there, why are you repeating it?

Comment (schmidthubert) [reply media-autobuild_suite.bat]: Oops, sorry, didn't see it, fixed now!

**Comment by wiiaboo** [comment]
Merged/fixed in de1de15.

**Pull request closed without merging**